sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonParser.java

```diff
@@ -43,6 +43,7 @@
             JsonValue value = Json.parse(reader);
             return parseJsonObject(value);
         } catch (Exception ioe) {
+            ioe.printStackTrace();
```

Comment:
> ioe.printStackTrace();

Likely is not needed. Assumption would be that whomever is invoking this method would handle either the exception or log it or something else.